app/src/main/java/com/hedvig/app/feature/offer/binders/PerilAdapter.kt

```diff
@@ -0,0 +1,64 @@
+package com.hedvig.app.feature.offer.binders
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.fragment.app.FragmentManager
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import com.hedvig.android.owldroid.graphql.OfferQuery
+import com.hedvig.app.BuildConfig
+import com.hedvig.app.R
+import com.hedvig.app.feature.offer.PerilBottomSheet
+import com.hedvig.app.util.extensions.isDarkThemeActive
+import com.hedvig.app.util.extensions.view.setHapticClickListener
+import kotlinx.android.synthetic.main.offer_peril_item.view.*
+
+class PerilAdapter(
```

Comment:
I think this one can also be merged with the one from `ContractCoverageActivity`